src/minio.d.ts

```diff
@@ -207,6 +207,9 @@ export class Client extends TypedClient {
   removeBucketTagging(bucketName: string, callback: NoResultCallback): void
```

Comment:
should remove these 2 methods and `setBucketTagging`

Reply:
Thank you. :+1: